### PR TITLE
NOTIF-504 Customize Splunk form

### PR DESCRIPTION
**Pull request by josejulio**
and shows it on prod-beta

Screenshot_2022-03-07_07-15-03

### CODE REVIEW

**Comment by josejulio** [comment]
@vkrizan could you take a look at the screenshot?

File: src/components/Integrations/Form/IntegrationTypeSplunkForm.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                ouiaId={ ouiaIdConcat(props.ouiaId, 'endpoint-url') }
+            />
+            <Checkbox
+                id="integration-type-camel-ssl-verification-enabled"
+                label="Enable SSL verification"
+                name="sslVerificationEnabled"
+                ouiaId={ ouiaIdConcat(props.ouiaId, 'is-ssl-verification-enabled') }
+            />
+            <FormGroup fieldId='integration-type-camel-secret-token'
+                helperText="The defined secret token is sent as a Splunk's HTTP Event Collector.">
```
Comment (vkrizan) [inline src/components/Integrations/Form/IntegrationTypeSplunkForm.tsx]: Maybe:
```
The defined secret token is sent as a Splunk's HTTP Event Collector token.
                                                                   ++++++
```

**Review by vkrizan — APPROVED** [review]
Looks great. I've added some suggestion to token's help text.

File: src/components/Integrations/Form/IntegrationTypeSplunkForm.tsx (hunk 1 above)
Comment (josejulio) [reply src/components/Integrations/Form/IntegrationTypeSplunkForm.tsx]:
```suggestion
                helperText="The defined secret token is sent as a Splunk's HTTP Event Collector token.">
```

Like this?

Comment (vkrizan) [reply src/components/Integrations/Form/IntegrationTypeSplunkForm.tsx]: Yes. Thanks.

**Pull request merged**